pkg/apis/networking/v1alpha1/clusteringress_defaults.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 const (
+	// MaxTimeout is the maximum allowed timeout for ClusterIngress.
+	MaxTimeout = 300 * time.Second
 	// DefaultTimeout will be set if timeout not specified.
 	DefaultTimeout = 60 * time.Second
```

Comment:
IMO the default should be the max and this should be toggled at the Revision level.